fs/dafsclient/rmtclient.cpp

```diff
@@ -1085,81 +1097,122 @@ ISocket *checkSocketSecure(ISocket *socket)
     return LINK(socket);
 }
 
-ISocket *connectDafs(SocketEndpoint &ep, unsigned timeoutms)
+ISocket *connectDafs(SocketEndpoint &ep, unsigned timeoutms, const IPropertyTree *service)
 {
     Owned<ISocket> socket;
 
-    if ( (securitySettings.queryConnectMethod() == SSLNone) || (securitySettings.queryConnectMethod() == SSLOnly) )
+    if (isContainerized())
```

Comment:
refactored, because containerized route is significantly simpler than convuluted bare-metal strategy of trying to connect to secure, then insecure etc.